src/popup/components/Modals/SecureLogin.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <Modal
+    class="secure-login"
+    from-bottom
+    full-screen
+    no-padding
+    centered
+    transparent
+    @close="handleClose"
+    @click="initAuthenticate"
+  >
+    <div
+      v-if="!isAuthenticated"
```

Comment:
Consider displaying something in case user is Authenticated.

Reply:
If user is Authenticated this modal should disappear

Reply:
OK, so why adding `v-if` for this case if other cases won't happen?